optaweb-employee-rostering/modules/optaweb-employee-rostering/install

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+SOURCES_DIR=/tmp/artifacts
+
+unzip -q ${SOURCES_DIR}/${ADD_ONS_DISTRIBUTION_ZIP} -d ${SOURCES_DIR}
+
+DISTRIBUTION_ZIP=${EMPLOYEE_ROSTERING_DISTRIBUTION_ZIP}
+DISTRIBUTION_WAR=${EMPLOYEE_ROSTERING_DISTRIBUTION_WAR}
+
+unzip -q ${SOURCES_DIR}/${DISTRIBUTION_ZIP} -d ${SOURCES_DIR}
+
+cp -rf ${SOURCES_DIR}/${DISTRIBUTION_WAR} ${JBOSS_HOME}/standalone/deployments/ROOT.war
+touch ${JBOSS_HOME}/standalone/deployments/ROOT.war.dodeploy
+#cp -rf ${SOURCES_DIR}/SecurityPolicy/* ${JBOSS_HOME}/bin/
```

Comment:
maybe remove it?

Reply:
@spolti I left the commented line in there because they should be adding that into their distribution.